vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -348,7 +350,7 @@ def set_relationship_on_hosts(hosts)
       #
 
       def normalize_blank_property_num(property)
-        property.try(:to_i)
+        property.try(:to_i) || 0
```

Comment:
:-) work == refresh will not fail

Right, in that case the normalize_methods should be rewritten to log warnings, when unexpected format will be there. I guess we should extract them to some common place, at least for OpenStack.